advanced/Scripts/query.sh

```diff
@@ -131,24 +114,26 @@ if [[ -n "${results[*]}" ]]; then
     done
 fi
 
-# Scan Wildcards
-if [[ -e "${wildcardlist}" ]]; then
-    # Determine all subdomains, domain and TLDs
-    mapfile -t wildcards <<< "$(processWildcards "${domainQuery}")"
-    for match in "${wildcards[@]}"; do
-        # Search wildcard list for matches
-        mapfile -t results <<< "$(scanList "${match}" "${wildcardlist}" "wc")"
-        if [[ -n "${results[*]}" ]]; then
-            if [[ -z "${wcMatch:-}" ]] && [[ -z "${blockpage}" ]]; then
-                wcMatch=true
-                echo " ${matchType^} found in ${COL_BOLD}Wildcards${COL_NC}:"
-            fi
-            case "${blockpage}" in
-                true ) echo "π ${wildcardlist##*/}"; exit 0;;
-                *    ) echo "   *.${match}";;
-            esac
+# Scan Regex
+if [[ -e "${regexlist}" ]]; then
+    mapfile -t results <<< "$(scanList "${domainQuery}" "${regexlist}" "rx")"
+    if [[ -n "${results[*]}" ]]; then
+        if [[ -z "${wcMatch:-}" ]] && [[ -z "${blockpage}" ]]; then
+            wcMatch=true
+            echo " ${matchType^} found in ${COL_BOLD}Regex list${COL_NC}:"
         fi
-    done
+
+        # Remove first full stop if it exists (e.g: ".foo.bar" > "foo.bar")
+        [[ "${results[0]::1}" == "." ]] && results[0]="${results[0]:1}"
+
+        # Return matching regex.list line (Full stops escaped to check against wildcards)
+        result=( "$(grep -i "${results[0]//./\\\\.}" "${regexlist}")" "$(grep -i "${results[0]}" "${regexlist}")" )
+
+        case "${blockpage}" in
+            true ) echo "π ${regexlist##*/}"; exit 0;;
+            *    ) echo "   ${result[*]}";;
```

Comment:
Only the first entry is indented:
```
$ pihole -q domain.co
 Match found in Regex list:
   .*domain\.com$
.*domain\.co
```

Reply:
Thanks! I tried reproducing this prior to the PR, but only one line would print.